etl/src/v2/workers/apply.rs

```diff
@@ -375,19 +324,20 @@ where
     type Error = ApplyWorkerHookError;
 
     async fn initialize(&self) -> Result<(), Self::Error> {
-        let table_replication_states = self.active_table_replication_states().await?;
-
-        for table_replication_state in table_replication_states {
-            let table_id = table_replication_state.table_id;
+        let table_replication_states = self.active_table_replication_states(true).await?;
 
+        for table_id in table_replication_states.keys() {
             let table_sync_worker_state = {
                 let pool = self.pool.read().await;
-                pool.get_active_worker_state(table_id)
+                pool.get_active_worker_state(*table_id)
             };
 
             if table_sync_worker_state.is_none() {
-                if let Err(err) = self.start_table_sync_worker(table_id).await {
-                    error!("Error handling syncing table {}: {}", table_id, err);
+                if let Err(err) = self.start_table_sync_worker(*table_id).await {
+                    error!(
+                        "Error starting table-sync worker for table {}: {}",
```

Comment:
Do we want to name them `table-sync` instead of `apply`?

Reply:
I updated the log line anyway.